.github/workflows/openhands.yml

```diff
@@ -1,19 +1,34 @@
-name: Resolve Issues with OpenHands
+name: Resolve Issue with OpenHands
 
 on:
   issues:
     types: [labeled]
+  pull_request:
+    types: [labeled]
+  issue_comment:
+    types: [created]
+
+permissions:
+  contents: write
+  pull-requests: write
+  issues: write
 
 jobs:
   call-openhands-resolver:
+    if: |
+      ${{
+        github.event.label.name == 'fix-me' ||
+        (github.event_name == 'issue_comment' && 
```

Comment:
@YeagerAI-Bulat could you please briefly explain what these conditions mean?

Reply:
This workflow will trigger if you add the label `fix-me` to an issue in this repo OR
when a user of YeagerAI (owner/collaborator/member) comments on an issue starting with a specific trigger phrase, for example mentioning `@openhands-agent` in the comments